setup/android_build_env.sh

```diff
@@ -10,9 +10,9 @@ LATEST_MAKE_VERSION="4.2.1"
 LATEST_CCACHE_VERSION="3.7.1+246_gd8e2ab9"
 LATEST_NINJA_VERSION="1.9.0.git"
 UBUNTU_14_PACKAGES="binutils-static curl figlet git-core libesd0-dev libwxgtk2.8-dev schedtool"
-UBUNTU_16_PACKAGES="libesd0-dev"
-UBUNTU_18_PACKAGES="curl"
-DEBIAN_10_PACKAGES="curl rsync"
+UBUNTU_16_PACKAGES="libesd0-dev git-core"
+UBUNTU_18_PACKAGES="curl git-core"
+DEBIAN_10_PACKAGES="curl rsync git-core"
```

Comment:
Just move this to common packages at this point

Reply:
makes sense and addressed